### DISCUSSION TITLE
As of latest relase 3.2 neotree buffer show line numbers when opened via an autocmd.

### DISCUSSION THREAD

**Discussion by croyleje** [nvim-neo-tree/neo-tree.nvim#1105] [General]
This appears to be related to commit 38293fe rolling back to 3.1 does fix the issue.  And setting a ftplugin config does not seem to fix it either seems like the Neotree window is being generated beofore the opts are loaded or something like that.  As soon as the Neotree window gets focus the number disapear.  Any one else see this?

**Comment by tranzystorekk** [comment]
This seems to be an unsolved problem even now on 3.27.0

(I have a simple autocmd config that loads my session and shows neotree on `VimEnter` and closes it on `VimLeavePre`)
